atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchEventCacheImpl.java

```diff
@@ -42,6 +44,8 @@
 
     private final LockWatchEventLog eventLog;
     private final TimestampStateStore timestampStateStore;
+    private final RateLimiter rateLimiter =
+            RateLimiter.create(1 / Duration.ofSeconds(5).getSeconds());
```

Comment:
was this because it's easier to test compared to using a random?

Reply:
also, how many permits is this supposed to be? Is this saying we can do it 0.2 times per second? 1 per second is likely plenty.

Reply:
Well, this one maintains a rate and the random way looks a bit jank too.

This is 1 every 5sec should I increase it to 1per sec?

Reply:
1 in 5 secs seems reasonable to me